apps/src/lib/node/ledger/shell/prepare_proposal.rs

```diff
@@ -141,6 +136,34 @@ where
     }
 }
 
+fn validate_tx_bytes(
```

Comment:
Could we change the name of this function to something like `check_expired`? If the jobs it performs in the future increases, we can move back to this vaguer name.

Reply:
This function also checks that the transaction is a `Wrapper`, that's why I chose a vague name

Reply:
Missed that. Ok then.